.travis.yml

```diff
@@ -87,8 +87,14 @@ before_install:
     - tools/check_sdist.py $SDIST_NAME
 
 install:
-    - python setup.py develop
     - ccache --show-stats
+    # Test installing without cython using the sdist
+    - if [[ $INSTALL_FROM_SDIST ]]; then
+        pip uninstall cython -y;
+        pip install dist/scikit-image-*.tar.gz;
+      else
+        pip install --no-build-isolation .;
```

Comment:
This flag requires a comment to explain why it is necessary, for future ourselves ;)